index.ts

```diff
@@ -163,34 +197,32 @@ export function autocomplete ({
  * @param  {string} [$0.url]
  * @return {Promise}                A Promise that'll get resolved with the bulk geocode result
  */
-export function bulk ({
+export function bulk({
   addresses,
+  boundary,
   clientId,
   clientSecret,
-  boundary,
   focusPoint,
   url
-}: BaseQuery & {
-  addresses: Array<any>,
-  boundary?: Boundary,
-  focusPoint?: any
-}): Promise<BaseResponse> {
-  const geocoder = getGeocoder(clientId, clientSecret, url)
-  let addressesQuery = [...addresses]
-  const options = {}
+}: BaseQuery & Options & { addresses: Array<string> }): Promise<
+  BaseResponse[]
+> {
```

Comment:
Two things:

1. Can you in-line the applicable keys within the `Options` type since not all of those values would apply as keys in the input argument for this function call?
2. Is the `Promise<BaseResponse[]>` return type correct? It seems it may be more like `Promise<BaseResponse & { query: Array<String> }>`.